packages/react/src/components/Cta/README.md

```diff
@@ -0,0 +1,54 @@
+# CTA
+
+> The CTA component will be used to select different cta types pages.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@import '@carbon/ibmdotcom-styles/scss/components/imagecomponent/imagecomponent';
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import 'yourapplication.scss';
+
+function App() {
+  return <CTA style={style} type={type} {...cta} />;
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the image styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+## Props
+
+| Name    | Required | Data Type | Default Value | Description                                    |
+| ------- | -------- | --------- | ------------- | ---------------------------------------------- |
+| `style` | NO       | String    | text          | Deafult style is text (LinkWithIcon component) |
+| `cta`   | NO       | Object    | null          | Contains type, href,copy title, heading, image |
+| `type`  | NO       | String    | local         | Contains type, href,copy title, heading, image |
```

Comment:
CTA isn't actually a prop, checkout the functional specs for these definitions. It adopts whatever the props are for the given CTA.

![image](https://user-images.githubusercontent.com/3793636/73770062-615aac00-4741-11ea-8ceb-bfa2ce46f619.png)

Reply:
so, we need add these {title, text, href} in  # props section of README.md  file?

Reply:
You might add a comment and or link to the other CTA components saying use their props here. I can try to add a more specific comment within the functional specs.

Reply:
Okay.